airflow/dag_processing/manager.py

```diff
@@ -498,7 +495,12 @@ def _refresh_dag_bundles(self):
 
             bundle_file_paths = self._find_files_in_bundle(bundle)
 
+            # Now that we have the files present in the latest bundle,
+            # we need to update the file paths to include any new ones
+            # and remove any that are no longer in the bundle.
+            # We do this by removing all existing files that are in this bundle
             new_file_paths = [f for f in self._file_paths if f.bundle_name != bundle.name]
+            # And then add all the current files in
```

Comment:
Incomplete sentence?

Reply:
It's a carry-on of the previous comment block. But I can reword it a bit, no problem.